docs/user/rstudio/ide/guide/ui/ui-panes.qmd

```diff
@@ -92,14 +92,11 @@ The Console pane also includes an integrated **Terminal** tab for executing syst
 
 The **Background Jobs** tab provides the ability to send long running R scripts to [local and remote background jobs](../tools/jobs.html). This functionality can dramatically improve the productivity of data scientists and analysts using R since they can continue working in RStudio while jobs are running in the background.
 
-<kbd>Ctrl+1</kbd> — Move focus to the Source Pane
-<kbd>Ctrl+2</kbd> — Move focus to the Console
-<kbd>Ctrl+L</kbd> — Clear the Console
-<kbd>Esc</kbd> — Interrupt R
+<kbd>Ctrl+1</kbd> --- Move focus to the Source Pane <kbd>Ctrl+2</kbd> --- Move focus to the Console <kbd>Ctrl+L</kbd> --- Clear the Console <kbd>Esc</kbd> --- Interrupt R
```

Comment:
Pane, Menu, Button, etc., shouldn't be capitalized unless it is labeled as such in the UI.
```suggestion
<kbd>Ctrl+1</kbd> --- Move focus to the Source pane <kbd>Ctrl+2</kbd> --- Move focus to the Console <kbd>Ctrl+L</kbd> --- Clear the Console <kbd>Esc</kbd> --- Interrupt R
```

Reply:
Did a global `Pane` to `pane` replacement. Thanks!